### PR TITLE
Add docs for max output option for apsystems integration

### PR DESCRIPTION
**Pull request by mawoka-myblock**
## Proposed change

Add docs regarding the new option to limit the output of an apsystems inverter

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/118825
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/apsystems.markdown [hunk 1]
```diff
@@ -3,22 +3,27 @@ title: APsystems
 description: Control and monitor your APsystems EZ1 microinverters locally without the cloud
 ha_iot_class: Local Polling
 ha_config_flow: true
-ha_release: 2024.6
+ha_release: 2024.7
```
Comment (frenck) [inline source/_integrations/apsystems.markdown]: This integration is released in 2024.6 right?

Comment (mawoka-myblock) [reply source/_integrations/apsystems.markdown]: No, afaik it will be released tomorrow.

Comment (frenck) [reply source/_integrations/apsystems.markdown]: Yup, 2024.6 is being released tomorrow...

Comment (mawoka-myblock) [reply source/_integrations/apsystems.markdown]: Oh, it's not my day today 🤦🏼 , you're right. Will fix that now.

**Comment by c0ffeeca7** [comment]
The PR comment says the ha_release is 2024.7, but I currently see 2024.6. Maybe this change has not been pushed yet?

**Comment by mawoka-myblock** [comment]
Ah, no. That's by mistake. The code still says 2024.6

**Comment by c0ffeeca7** [comment]
> Ah, no. That's by mistake. The code still says 2024.6

What do you mean by: the code says 2024.6? If the core code has only been merged now and has not been tagged for a milestone, it will go into the 2024.7. And the docs should then state 2024.7.

**Comment by mawoka-myblock** [comment]
Ah, interesting, did not know that!

**Comment by mawoka-myblock** [comment]
But the integration is already merged, I just added a new feature to it.

**Comment by c0ffeeca7** [comment]
> But the integration is already merged, I just added a new feature to it.

oh, right. my mistake.

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @mawoka-myblock :+1:

**Pull request merged**